experiment/generate_tests.py

```diff
@@ -281,7 +270,7 @@ def for_each_job(output_dir, job_name, job, yaml_config):
         generator = E2ENodeTest(job_name, job, yaml_config)
     else:
         raise ValueError('Unexpected job type ', job_type)
-    envs, job_config, prow_config = generator.generate()
+    job_config, prow_config = generator.generate()
```

Comment:
this doesn't look like update bazel? /shrug guess the commits don't matter too much

Reply:
I probably amended into that commit :-)